src/linear_samplers/block_col_gaussian.jl

```diff
@@ -0,0 +1,46 @@
+# This code was written by Nathaniel Pritchard
+"""
+    LinSysBlkColGaussSampler <: LinSysBlkColSampler 
+
+A mutable structure with fields to handle Guassian column sketching. 
+
+# Fields
+- `blockSize::Int64` - Specifies the size of each block.
+- `sketchMatrix::Union{AbstractMatrix, Nothing}` - The buffer for storing the Gaussian sketching matrix.
+- `scaling::Float64` - The variance of the sketch, is set to be s/n.
```

Comment:
What is `s`?

Reply:
It should be the sketch size. This should have been fixed clarified in more recent commits.